src/plugins/hosts/README.md

```diff
@@ -83,7 +83,7 @@ sudo kdb mount --with-recommends hosts /tests/hosts hosts
 mkdir -p $(dirname `kdb file user:/tests/hosts`)
 
 # Create hosts file for testing
-echo '127.0.0.1	localhost' >  `kdb file user:/tests/hosts`
+echo '127.0.0.1	localhost' >>  `kdb file user:/tests/hosts`
```

Comment:
According to the comment above I think this was okay, `>` creates/truncates the file (>> also creates but appends if present).